support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -150,15 +160,30 @@ class CreateSubscriptionController(
         }
     }
 
-  private def logIncomingRequest(
-      request: OptionalAuthRequest[CreateSupportWorkersRequest],
-      maybeLoggedInIdentityIdAndEmail: Option[IdentityIdAndEmail],
-  ) = {
+  private def errorDetailedMessage(message: String)(implicit request: CreateRequest) = {
```

Comment:
these two methods are the same apart from error vs  info?  If so consider to just make one method
e.g. one that takes a `String => Unit` as a parameter and pass in either SafeLogger.error or SafeLogger.info as the parameter

Reply:
I thought about it - but it felt clearer and easier to change in future to just copy/paste until I had another usecase.